elastic/kibana
# [AO] Implement Alert Summary Widget new design

Closes #149239
Closes #149238

## 📝 Summary

This PR implements the new design of the Alert Summary Widget. In the new design, we removed the recovered chart and count to make it easier to understand. ([design](https://www.figma.com/file/xnSsLoEMntX3VLG0qwmmnt/Alert-summary-widget-V2?t=m2Xd16Obz5OJz7A1-0)) After discussion with @maciejforcone, we decided to use blue color instead of black since when we have one alert that triggers the whole time during the selected period, the chart was not readable.

|Full-size|Compact|
|---|---|
|![image](https://user-images.githubusercontent.com/12370520/214273219-639ccb58-8092-4f93-95d6-8faf8f3791bd.png)|![image](https://user-images.githubusercontent.com/12370520/214273271-4d4f3ba4-196a-472a-903e-913583f2678f.png)|

https://user-images.githubusercontent.com/12370520/214258610-2d5d0b9b-9034-4cec-885f-c57959cd7d53.mov

## 🧪 How to test
- Check the component's new design in [storybook](https://ci-artifacts.kibana.dev/storybooks/pr-149348/5181917c254d5a4e6038be7ceb5e551fcab03161/triggers_actions_ui/index.html?path=/story/app-alertsummarywidget--compact)
- Generate some alerts
  - Check the Alerts page; you should see the Alert Summary Widget show the correct data
  - Check the Rule details of one of the alerts to see the compact version there
  - Clicking on Alert Summary Widget compact version should work as expected
